cmd/libsnap-confine-private/error.h

```diff
@@ -89,6 +89,26 @@ __attribute__((warn_unused_result,
 	       format(printf, 3, 4) SC_APPEND_RETURNS_NONNULL))
 sc_error *sc_error_init(const char *domain, int code, const char *msgfmt, ...);
 
+/**
+ * Initialize an unspecified error with formatted message.
+ *
+ * This is just syntactic sugar for sc_error_init(SC_LIBSNAP_ERROR,
+ * SC_UNSPECIFIED_ERROR, msgfmt, ...) which is repeated often.
+ **/
+__attribute__((warn_unused_result,
+	       format(printf, 1, 2) SC_APPEND_RETURNS_NONNULL))
+sc_error *sc_error_init_simple(const char *msgfmt, ...);
+
+/**
+ * Initialize an unspecified error with formatted message.
```

Comment:
`Initialize an api misuse error with formatted message.` ?

Reply:
```suggestion
 * Initialize an api misuse error with formatted message.
```

Reply:
s/api/API/ even in @anonymouse64's suggestion